core/domain/feedback_services.py

```diff
@@ -243,13 +242,10 @@ def add_message_id_to_read_by_list(thread_id, user_id, message_id):
         message_id: int. The id of the message.
     """
     feedback_thread_user_model = (
-        feedback_models.GeneralFeedbackThreadUserModel.get(user_id, thread_id))
-
-    if not feedback_thread_user_model:
-        feedback_thread_user_model = (
-            feedback_models.GeneralFeedbackThreadUserModel.create(
-                user_id, thread_id))
-
+        feedback_models.GeneralFeedbackThreadUserModel.get(
```

Comment:
Just curious how does this work? wouldn't this evaluate to a boolean?

Reply:
Ah no, in Python the `and` and `or` operators always return one of the operands, so `a and b` is exactly equivalent to `b if a else a`, and `a or b` is exactly equivalent to `a if a else b`. Short-circuiting still happens, though, so `b` wouldn't get evaluated if `a` is the operand that would be returned.